### PR TITLE
Add otfftw recipe

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/otfftw/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+if "%ARCH%"=="32" (set CPU_ARCH=i686) else (set CPU_ARCH=x86_64)
+%PKG_NAME%-%PKG_VERSION%-py%PY_VER%-%CPU_ARCH%.exe /userlevel=1 /S /FORCE /D=%PREFIX%
```
Comment (jakirkham) [inline recipes/otfftw/bld.bat]: Can we not build this on Windows too? Seems like we are using `cmake`, which should make this very easy.

Comment (jschueller) [inline recipes/otfftw/bld.bat]: @jakirkham the problem is building openturns with msvc; we only succeeded on building for vc10, without bindings. Also the build time would be a problem.

Comment (jakirkham) [inline recipes/otfftw/bld.bat]: So how are these binaries getting built then?

Comment (jschueller) [inline recipes/otfftw/bld.bat]: MinGW, from Linux.

**Comment by jschueller** [comment]
@jakirkham is it ok to be deployed ?

**Comment by ocefpaf** [comment]
@jschueller you just need to keep track of the Windows re-packing. I recommend opening issues on the feedstock.

Thanks for the PR!

**Pull request merged**